understanding/on-focus.html

```diff
@@ -55,88 +53,77 @@
                   
-                  <dt>Goal</dt>
-                  <dd>Content can be navigated more predictably.</dd>
+                  <dt>目標</dt>
+                  <dd>コンテンツをより予測可能な方法で操作できる。</dd>
                   
-                  <dt>What to do</dt>
-                  <dd>Do not change a user's context when items get focus.</dd>
+                  <dt>何をすればよいか</dt>
+                  <dd>項目にフォーカスが当たったときに、利用者のコンテキストを変更しない。</dd>
                   
-                  <dt>Why it's important</dt>
-                  <dd>Content that behaves predictably is especially important to people with disabilities.</dd> 
+                  <dt>なぜそれが重要か</dt>
+                  <dd>予測可能な動作をするコンテンツは、障害のある人にとって特に重要である。</dd> 
                   
                </dl>
             </section>
             <section id="intent">
-               <h2>Intent</h2>
-               <p>The intent of this Success Criterion is to ensure that functionality is predictable
-                  as visitors navigate their way through a document. Any component that is able to trigger
-                  an event when it receives focus must not change the context.          Examples of
-                  changing context when a component receives focus include, but are not limited to:
+               <h2>意図</h2>
+               <p>この達成基準の意図は、訪問者がドキュメント内をナビゲートする際に、機能が予測可能であることを保証することである。フォーカスを受け取ったときにイベントを起動することのできるすべてのコンポーネントは、コンテキストを変化させてはならない。コンポーネントがフォーカスを受け取ったときにコンテキストを変更する例には、次のものがあるが、これに限定されない:
                   
                   
                </p>
                <ul>
                   
                   
-                  <li>forms submitted automatically when a component receives focus;</li>
+                  <li>コンポーネントがフォーカスを受け取るときにフォームが自動的に送信される。</li>
                   
                   
-                  <li>new windows launched when a component receives focus;</li>
+                  <li>コンポーネントがフォーカスを受け取るときに新しいウィンドウを開く。</li>
                   
                   
-                  <li>focus is changed to another component when that component receives focus;</li>
+                  <li>コンポーネントがフォーカスを受け取るときにフォーカスが別のコンポーネントに変更される。</li>
                   
                   
                </ul>
-               <p>Focus may be moved to a control either via the keyboard (e.g. tabbing to a control)
-                  or the mouse (e.g. clicking on a text field). Moving the mouse over a control does
-                  not move the focus
-                  unless scripting implements this behavior. Note that for some types of controls, clicking
-                  on a control may also activate the control (e.g. button), which may, in turn, initiate
-                  a change in context.
+               <p>フォーカスはキーボード操作 (例: コントロールにタブ移動する) 又はマウス操作 (例: テキストフィールドをクリックする) のいずれかを介してコントロールに移動させてもよい。スクリプトがこの動作を実装しない限り、コントロール上にマウスを移動しても、フォーカスは移動しない。順番にコンテキストの変更を開始することのある一部の種類のコントロールでは、コントロールをクリックするとそのコントロール (例: ボタン) もアクティブにする場合があることに注意されたい。
```

Comment:
後半は誤訳かも。ここの in turn は「順番に」というより「それによって」「その結果」という意味かと思います。コントロールをクリックするとそのコントロールにフォーカスがうつるが、ボタンなどの場合はフォーカスが移るだけでなくボタンを押す効果も発生してコンテキストが変化する場合がある、という話ですね。

may also activate あたりのニュアンスがなんとも日本語にしづらいですが、たとえばこんな感じでしょうか?

```suggestion
               <p>フォーカスはキーボード操作 (例: コントロールにタブ移動する) 又はマウス操作 (例: テキストフィールドをクリックする) のいずれかを介してコントロールに移動させてもよい。スクリプトがこの動作を実装しない限り、コントロール上にマウスを移動しても、フォーカスは移動しない。ある種のコントロールにおいては、コントロールをクリックすることが、そのコントロールをアクティブにする効果も持つことがあり (例: ボタン)、その結果として、コンテキストの変化が始まるかもしれないことに注意されたい。
```